.github/workflows/release.yaml

```diff
@@ -0,0 +1,43 @@
+name: release
+
+on:
+  push:
+    branches:
+      - '**'
+    tags:
+      - 'v*'
+  pull_request:
+    branches:
+    - main
+
+jobs:
+  goreleaser:
+    name: goreleaser
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ~1.17
+
+    - name: Docker Login
+      uses: docker/login-action@v1
+      if: startsWith(github.ref, 'refs/tags')
+      with:
+        registry: quay.io
+        username: ${{ secrets.QUAY_USERNAME }}
+        password: ${{ secrets.QUAY_PASSWORD }}
+
+    - name: Run goreleaser
+      run: |
+        if [[ $GITHUB_REF == refs/tags/* ]]; then
+          export GORELEASER_ARGS="--rm-dist"
```

Comment:
It looks like we pass the `--rm-dist` argument in the Makefile as well -- should we be passing it twice?

Reply:
So we're actually overriding the default GORELEASER_ARGS variable, and _removing_ the --snapshot option. It's not the most readable (or obvious) implementation, so I can understand any potential confusion.